config/routes.rb

```diff
@@ -718,22 +710,31 @@
 
   get 'admin/mailer_previews/*path' => "rails/mailers#preview", as: :admin_mailer_preview
 
-  #redirect from old teacher dashboard
-  namespace :teachers do
-    get '/schools/:name', to: redirect('/management/schools/%{name}')
-  end
-
-  namespace :management do
-    resources :schools, only: :show do
-      resources :management_priorities, only: :index
-    end
-  end
-
   namespace :pupils do
     resource :session, only: [:create]
     resources :schools, only: :show do
       get :analysis, to: 'analysis#index'
       get 'analysis/:energy/:presentation(/:secondary_presentation)', to: 'analysis#show', as: :analysis_tab
     end
   end
+
+  # LEGACY PATHS
+  # Old paths that (may) have been indexed by crawlers or used in comms/emails that
+  # we want to maintain and redirect.
+
+  # Old 'find out more' pages
+  get '/schools/:name/find_out_more', to: redirect('/schools/%{name}/advice', status: 301)
```

Comment:
think 301 is the default?

Reply:
Thanks. I was getting mixed up with `redirect_to` which defaults to 302 instead so where these redirects where done in controllers previously, we had to set status.